README.md

```diff
@@ -241,7 +238,6 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Remote Report](http://remote.report) - A collection of the best remote, support focused and non-technical jobs delivered weekly
   1. [Remoteur](http://www.remoteur.com) - Remote jobs in Europe delivered to your inbox bi-weekly
   1. [Remotive - productive remote workers](http://remotive.io/) - A weekly newsletter on Remote Tips & Jobs sent to 10,000+ Remote Workers
-  1. [Working Nomads](http://workingnomads.co/) - Job posts

```

Comment:
this one works as well (301)

Reply:
Ahh, yes, I think I removed this one as a duplicate / miscategorized. Sorry, should have done in a separate PR.

Reply:
Want me to leave this one in and resubmit?